python/src/RandomVectorMetropolisHastings_doc.i.in

```diff
@@ -5,63 +5,82 @@ Refer to :ref:`bayesian_calibration`, :ref:`metropolis_hastings`.
 
 Parameters
 ----------
-source : :class:`~openturns.RandomVector`
-    Source term
+randomVector : :class:`~openturns.RandomVector`
+    The random variable from which to update the chain
 initialState : sequence of float
     Initial state of the chain
 marginalIndices : sequence of int, optional
     Indices of the components to be updated.
     If not specified, all components are updated.
     The number of updated components must be equal to the dimension of *proposal*.
-sourceLinkFunction : :class:`~openturns.Function`, optional
-    Link between the state of the chain and the parameters of *source*
+linkFunction : :class:`~openturns.Function`, optional
+    Link between the state of the chain and the parameters of *randomVector*.
+    If not provided, then the parameters of *randomVector* are not updated,
+    which means that samples from *randomVector* are produced independently
+    from the state of the Markov chain.
 
 Notes
 -----
-The random walk Metropolis-Hastings algorithm is
-a Markov Chain Monte-Carlo algorithm.
-It draws candidates for the next state of the chain following direct
-realizations of the *source* random variable.
+This class creates a Markov chain from a :class:`~openturns.RandomVector`.
+It updates the parameters of the random vector
+based on the current state of the Markov chain,
+gets a realization from the random vector with the updated parameters,
+and then uses it to update the Markov chain.
+
+If no likelihood is set with the :meth:`setLikelihood` method,
+then it can be viewed as trivial Metropolis-Hastings algorithm
+where the proposal distribution is equal to the target distribution,
+so the Metropolis-Hastings ratio is always equal to 1
+and the candidate is always accepted.
+
+If a likelihood is set, then the Metropolis-Hastings ratio
+is the ratio of the likelihoods of the new and of the current state.
+
+See Also
+--------
+Gibbs, RandomWalkMetropolisHastings
 
 Examples
 --------
 >>> import openturns as ot
 >>> import math as m
 >>> ot.RandomGenerator.SetSeed(0)
```

Comment:
```suggestion
```
Do we really need those lines?

Reply:
I don't know for the first one, but for the second one it is mandatory if you want to have a robust output, otherwise if another class using RandomGenerator is inserted in the documentation before RandomVectorMetropolisHastings, then the state of RandomGenerator will be different when the snippets of code will be executed, producing a different output than the reference one.

Reply:
No output is printed, so I don't think it matters.

Reply:
the math import is useless, but lets avoid surprises with the state of the rng

Reply:
Works for me.